cgi-bin/DW/Controller/Media.pm

```diff
@@ -39,7 +39,17 @@ sub media_manage_handler {
 
     # load all of a user's media.  this is inefficient and won't be like this forever,
     # but it's simple for now...
-    $rv->{media} = [ DW::Media->get_active_for_user( $rv->{remote} ) ];
+    my @media = DW::Media->get_active_for_user( $rv->{remote}, width => 200, height => 200 );
+    my @valid_sizes;
+    map { push @valid_sizes, ( $_, $_ ) } keys %VALID_SIZES;
+
+    $rv->{media} = \@media;
+    $rv->{make_embed_url} = \&make_embed_url;
+    $rv->{page} = $rv->{r}->get_args->{page} || '1';
+    $rv->{view_type} = $rv->{r}->get_args->{view} || '';
+    $rv->{maxpage} = POSIX::ceil(scalar @media / 20);
```

Comment:
When using a function from a module, it's best practice to make sure the module is included in your file, unless you know that one of the modules you're already explicitly including also uses it.  So make sure you have `use POSIX;` above, or possibly `use POSIX qw( ceil );` if you want to be specific about what functions are imported.